components/chart/canvas/ModulesCanvas.tsx

```diff
@@ -20,6 +21,10 @@ function ModulesCanvas({
     <>
       {Object.keys(currentVersion.nodes).map((moduleKey) => {
         const module = currentVersion.nodes[moduleKey];
+        useEffect(() => {
+          module.x += DEFAULT_X_PADDING;
+          module.y += DEFAULT_Y_PADDING;
+        }, []);
```

Comment:
I think also that would cause performance issues, I think the best place to add the default padding in
packages\hychart\components\chart\parseContentsToNodes.ts
when mapping DConfig data to nodes data, we can add default values there

Reply:
I'm sorry, I understood that I have to put it in a file with the module rendering.
Done